src/main/java/uk/gov/companieshouse/certifiedcopies/orders/api/validator/RequestValidator.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.companieshouse.certifiedcopies.orders.api.validator;
+
+import uk.gov.companieshouse.certifiedcopies.orders.api.dto.CertifiedCopyItemOptionsRequestDTO;
+import uk.gov.companieshouse.certifiedcopies.orders.api.logging.LoggingUtils;
+import uk.gov.companieshouse.certifiedcopies.orders.api.model.CertifiedCopyItemOptions;
+import uk.gov.companieshouse.certifiedcopies.orders.api.model.DeliveryMethod;
+import uk.gov.companieshouse.certifiedcopies.orders.api.util.FieldNameConverter;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+import java.beans.PropertyDescriptor;
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.lang.Boolean.TRUE;
+import static java.util.Arrays.stream;
+import static org.apache.commons.lang.StringUtils.isBlank;
```

Comment:
Sonar report indicates some of these imports are unused. Can the unused imports be removed please?

Reply:
These have been removed.